Makefile

```diff
@@ -125,6 +127,10 @@ release-install:
 	env CGO_ENABLED=0 $(GOINSTALL) -v -trimpath -ldflags="$(RELEASE_LDFLAGS)" -tags="$(RELEASE_TAGS)" $(PKG)/cmd/lnd
 	env CGO_ENABLED=0 $(GOINSTALL) -v -trimpath -ldflags="$(RELEASE_LDFLAGS)" -tags="$(RELEASE_TAGS)" $(PKG)/cmd/lncli
 
+protolint-install:
+	@$(call print, "Installing protolint.")
+	scripts/install_protolint.sh $(PROTOLINT_DOWNLOAD_DIR)
```

Comment:
Since we use docker for all other CI steps that require extra software to be installed, I think we should use the maintained image described here: https://github.com/yoheimuta/protolint#use-the-maintained-docker-image